### PR TITLE
Add new text field screen

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Create new text field page Flutter Gallery App
## Task
Link of task: https://www.notion.so/dustin0306/TextField-5d29d8cf6a244e13ba600c7747a166a7?pvs=4
## Screenshot
image

## Videos

https://github.com/krish-do-goldenowl/walter-theory-trainings/assets/144195318/890a096e-9c87-4b5a-8740-060d97ed5713

### CODE REVIEW

**Comment by dustin-goldenowl** [comment]
In video record, I have observed that the text input is obscured by the keyboard. Please make changes so that the text input is visible above the keyboard.

**Comment by WalterGoldenOwl** [comment]
Hello @dustin-goldenowl I have resolved the problem in new commit

https://github.com/krish-do-goldenowl/walter-theory-trainings/assets/144195318/fd7d9442-b366-43cc-b82b-3188fdd6c46d

File: Flutter/flutter_gallery_app/lib/src/presentation/text_field/screen/text_field_page.dart [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider<TextFieldBloc>(
+      create: (context) => TextFieldBloc(),
+      child: const TextFieldView(),
+    );
+  }
+}
+
+class TextFieldView extends StatefulWidget {
```
Comment (dustin-goldenowl) [inline Flutter/flutter_gallery_app/lib/src/presentation/text_field/screen/text_field_page.dart]: You should separate the 'TextFieldView' class into a separate file. And then change name "TextFieldView" to "TextFieldWidget"

Comment (WalterGoldenOwl) [reply Flutter/flutter_gallery_app/lib/src/presentation/text_field/screen/text_field_page.dart]: Thank you, I have resolved the problem in new commit

**Pull request merged**